### ISSUE TITLE
SA1612 triggered by using reserved keywords as parameter names

### ISSUE THREAD

**Issue by BennieCopeland** [DotNetAnalyzers/StyleCopAnalyzers#2280]
The check ```SA1612: The parameter '{param name}' does not exist.``` is being triggered by parameters that use the prefixed keywords for parameters. I'm using the 1.1.0-beta001 version of the analyzer.

```c#
/// <summary>
/// SomeMethod summary
/// </summary>
/// <param name="event">Does something</param>
private void SomeMethod(string @event)
{
}
```

**Comment by bjornhellander** [comment]
OK. Thanks for the report!
For now, you can suppress the diagnostic and we will have a look at why this is happening.

**Comment by bjornhellander** [comment]
I'm fixing this. And I will try to see if there are similar problems with other rules.

**Comment by bjornhellander** [comment]
For the record, the problem was that the rule tried to get the parameter name through parameter.Identifier.Text, but that property contains the actual text written in the code, which in this case is "@event". On the other hand, parameter.Identifier.ValueText contains just "event", so I changed the code to use that property instead.
